mpl_infer_tiles_GPU_new.py

```diff
@@ -69,146 +78,179 @@ def run(self):
         # Useful if you're training a model on the same
         # machine, in which case use CPU and leave the
         # GPU for training.
-        DEVICE = "/gpu:%s"%(self.gpu_id)  # /cpu:0 or /gpu:0
-        os.environ['CUDA_VISIBLE_DEVICES'] = "{}".format(self.gpu_id)
+        if self.use_gpu:
+            os.environ["CUDA_VISIBLE_DEVICES"] = "{}".format(self.process_counter)
```

Comment:
nit could add a comment about what a process counter is?

Reply:
added the comment in the constructor where we set the property.